_data/food.yml

```diff
@@ -23,6 +23,13 @@ websites:
     facebook: grubhub
     twitter: Grubhub
 
+  - name: Lieferando
+    url: https://www.lieferando.de/en/
+    img: lieferando.png
+    facebook: lieferando
+    twitter: lieferando
+    email_address: info@lieferando.de
```

Comment:
I don't see any mention of this email address on their site.

Reply:
It's on their impressum page (which is in German even on the English site): https://www.lieferando.de/en/colofon